File: Unidad 1/T1.1 SIN BD/TI1.1 SOAP_DOTNET_CONUNI_GR01/02 CLICON/02. CLICON/Controller/AuthController.cs
using _02.CLICON.Models;
using _02.CLICON.Views;

namespace _02.CLICON.Controllers
{
    public class AuthController
    {
        // Login “quemado”
        private const string USER = "MONSTER";
        private const string PASS = "MONSTER9";

        private readonly ConsoleView _view;
        private readonly UserSession _session;

        public AuthController(ConsoleView view, UserSession session)
        {
            _view = view; _session = session;
        }

        public bool Login()
        {
            var (u, p) = _view.AskCredentials();
            var ok = (u == USER && p == PASS);
            if (ok) _session.SignIn(u);
            _view.ShowLoginResult(ok);
            return ok;
        }
    }
}

File: Unidad 1/T1.1 SIN BD/TI1.1 REST_JAVA_CONUNI_GR01/02 CLIMOV/02 CLIMOV/app/src/main/java/com/example/a02climov/controller/AuthController.kt
package com.example.a02climov.controller

import com.example.a02climov.model.Session

object AuthController {
    private const val USER = "MONSTER"
    private const val PASS = "MONSTER9"

    fun login(u:String, p:String): Session? =
        if (u == USER && p == PASS) Session(u) else null
}
